test/certs/root-ed25519.privkey.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+MC4CAQAwBQYDK2VwBCIEINTuctv5E1hK1bbY8fdp+K06/nwoy/HU++CXqI9EdVhC
+-----END PRIVATE KEY-----
```

Comment:
Why was root-ed25519.privkey.pem added as part of this pull request? Was it by accident?

Reply:
Uhmmmm...  yeah, I think so